docs/competition/event/incident-management.md

```diff
@@ -0,0 +1,85 @@
+# SR2019 Incident Management Procedure
+
+This page is primarily for internal use. If you are a team competing in Student Robotics, you probably want [our main website](https://studentrobotics.org/).
+
+This page details the procedures to be following in the event of an incident occuring during the competition. Incidents are split into 2 types; minor and major.
+
+What constitutes an incident is rather unpredictable, and as such is not fully defined. Common sense should prevail in determining the nature and severity of the incident.
+
+This procedure is superceded by any requirements of the venue, e.g. in the event of a fire follow the venues' fire evacuation procedure.
+
+# Minor Incidents
+
+A minor incident is anything that needs some kind of response but has minimal impact on the event.
+
+Examples may be;
+
+* A competitor has twisted their ankle on a stairwell,
+* Infrastructure cabling has become damaged and poses a hazard.
+
+Procedure here is simple;
+
+* Notify relevant coordinator,
+* Follow their instructions
+
+# Major Incidents
+
+A major incident is anything that causes a significant impact on the event as a whole.
+
+Examples might include;
+
+* Accident requiring significant first aid,
+* Battery failure renders access to an area unsafe
+
+As with minor incidents, the initial reaction should be to notify the relevant coordinator and follow their instructions. If first aid is required the First Aid Provider should __always__ be notified first.
```

Comment:
nit: maybe put the first aid part first, so that anyone reading this while responding to an incident (hopefully won't happen though I suspect people's order of consumption of information reflects how they use it) are prompted to seek first aid first?

```suggestion
If first aid is required the First Aid Provider should __always__ be notified first.
Following that, as with minor incidents, the reaction should be to notify the relevant coordinator and follow their instructions. 
```